app/config/collections.php

```diff
@@ -2711,29 +3339,51 @@
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('status'),
+                'array' => false,
+                '$id' => ID::custom('method'),
                 'type' => Database::VAR_STRING,
```

Comment:
Can this attribute be a constant?

Reply:
We will refactor all this kind of ENUMs like fields as a separate PR.